siriuspy/siriuspy/cycle/fc_cycle_data.py

```diff
@@ -0,0 +1,187 @@
+"""SI fast correctors cycle data."""
+
+import numpy as _np
+
+
+PARMS = {
```

Comment:
@anacso17 why did we choose to put this table here and not on CS-constants?

Reply:
I don't have a strong opinion about it. I just asked because I thought all parameters for cycling were at the control-system-constants repository. Since Linac parameters are already here, maybe we could leave these ones here too...

Reply:
I would also expect it to be in control-system-constants. maybe we have move it ove rthere later...